File: server/AppSupport.Data/Entities/Organization.cs
using System;
using System.Collections.Generic;

namespace AppSupport.Data.Entities
{
    public class Organization
    {
        public int Id { get; set; }
        public string Name { get; set; }

        public IEnumerable<Manifest> Manifests { get; set; }
        public IEnumerable<ManifestPerson> ManifestPeople { get; set; }
        public IEnumerable<Person> People { get; set; }
        public IEnumerable<Plane> Planes { get; set; }
        public IEnumerable<Template> Templates { get; set; }
    }
}

File: client/core/api/models/organization.ts
import {
  FormBuilder,
  FormGroup,
  Validators
} from '@angular/forms';

import { Manifest } from './manifest';
import { ManifestPerson } from './manifest-person';
import { Person } from './person';
import { Plane } from './plane';
import { Template } from './template';

export interface Organization {
  id: number;
  name: string;

  manifests: Manifest[];
  manifestPeople: ManifestPerson[];
  people: Person[];
  planes: Plane[];
  templates: Template[];
}

export const OrganizationForm = (org: Organization, fb: FormBuilder): FormGroup =>
  fb.group({
    id: [org.id],
    name: [org.name, Validators.required]
  });
